mmdet/ops/sigmoid_focal_loss/functions/sigmoid_focal_loss.py

```diff
@@ -0,0 +1,34 @@
+from torch.autograd import Function
+from torch.autograd.function import once_differentiable
+
+from .. import sigmoid_focal_loss
+
+
+class _SigmoidFocalLoss(Function):
```

Comment:
We can just follow the naming convention of this repo to name it as `SigmoidFocalLossFunction`.